### ISSUE TITLE
Add builtin rule `no_swallow_error`

### ISSUE THREAD

**Issue by KurtWagner** [KurtWagner/zlinter#42]
Maybe needs some more thought but 

```
something() catch {};
something() catch unreachable;
// to
something() catch |err| {
    std.log.err("unexpected error: {}\n", .{err});
    return err;
}
```
